src/Splunk.Logging.Common/HttpEventCollectorSender.cs

```diff
@@ -243,24 +262,18 @@ public enum SendMode
         private void DoSerialization(HttpEventCollectorEventInfo ei)
         {
             
-            string serializedEventInfo;
-            if (formatter == null)
-            {
-                serializedEventInfo = SerializeEventInfo(ei);
-            }
-            else
+            if (formatter != null)
```

Comment:
make it possible for DE and NL locales to get data through this call.

Reply:
I can see you have a PR already to do this. This one should stay focused on improving memory usage, and your PR can provide DE / NL locale support.